File: java_app/JavaGreenCloud/src/main/java/functions/PrimeNumbers.java
package functions;
import java.util.ArrayList;
import java.util.List;

public class PrimeNumbers {

    public static List<Integer> primeNumbers(int n) {

        List<Integer> result = new ArrayList<>();

        for (int i = 2; i <= n; i++) {
            if (isPrime(i)) {
                result.add(i);
            }
        }

        return result;
    }

    public static boolean isPrime(int n) {

        if (n < 2) {
            return false;
        }

        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }

        return true;
    }

    public static int sumPrimeNumbers(int n) {
        List<Integer> primes = primeNumbers(n);
        int sum = 0;

        for (int prime : primes) {
            sum += prime;
        }

        return sum;
    }
}

File: app1/src/prime_numbers.py
# Corrected prime_numbers function
def prime_numbers(n):
    def is_prime(n):
        if n < 2:
            return False
        for i in range(2, int(n ** 0.5) + 1):
            if n % i == 0:
                return False
        return True

    result = []
    for i in range(n + 1):
        if is_prime(i):
            result.append(i)
    return result

# sum_prime_numbers function
def sum_prime_numbers(n):
    return sum(prime_numbers(n))

